control_panel_api/services.py

```diff
@@ -153,6 +153,17 @@ def delete_bucket_policies(bucket_name):
     aws.delete_policy(policy_arn_readonly)
 
 
+def apps3bucket_create(apps3bucket):
+    app_slug = apps3bucket.app.slug
+    bucket_name = apps3bucket.s3bucket.name
+    readwrite = (apps3bucket.access_level == READWRITE)
```

Comment:
Parentheses not needed

Reply:
Mmmmm...but I like parenthesis.

Alright. I'll remove them then.

Reply:
https://github.com/ministryofjustice/analytics-platform-control-panel/pull/23/commits/a0d0bef0383bc7947237a10c6989482ee252f302